circuit/pallets/execution-delivery/src/lib.rs

```diff
@@ -553,16 +553,17 @@ pub mod pallet {
             gateway_genesis: GatewayGenesisConfig,
             first_header: Vec<u8>,
             authorities: Option<Vec<T::AccountId>>,
+            allowed_side_effects: Option<Vec<Vec<u8>>>,
```

Comment:
Introduce a type AllowedSideEffect = `Vec<u8>` and use consistently. 

Otherwise, looking good :)

Reply:
Not sure about Option here, since we introduce this list, why not make it required. If not, what does 'None' mean on this field?

Reply:
Removed option and added new type alias.